test/test_reductions.py

```diff
@@ -566,13 +566,13 @@ def get_values(x):
                         self.assertEqual(i, index)
                 self.assertEqual(torchfn(x), nan)
 
-    @dtypesIfCPU(torch.float, torch.double, torch.long, torch.bool)
+    @dtypesIfCPU(torch.float, torch.double, torch.long, torch.bool, torch.half)
```

Comment:
These tests can't be enabled for `bfloat16`, as they use `numpy` as reference, but it doesn't currently support `bfloat16`.

As a sidenote, I noticed that these tests haven't been enabled for `double` on CUDA, although they pass on CUDA.